dumpling/dumpling.go

```diff
@@ -91,6 +91,10 @@ func (m *Dumpling) Process(ctx context.Context, pr chan pb.ProcessResult) {
 	failpoint.Inject("dumpUnitProcessForever", func() {
 		m.logger.Info("dump unit runs forever", zap.String("failpoint", "dumpUnitProcessForever"))
 		<-ctx.Done()
+		pr <- pb.ProcessResult{
```

Comment:
maybe ProcessForever means no ProcessResult is sent @lichunzhu

Reply:
i think it ok to send result to pr chan ,beacuse another use of this failpint is [dm-worker-test.TestTaskAutoResume](https://github.com/pingcap/dm/blob/3cb459a17c24d1cb8d506c9a7b5094a5dacab717/dm/worker/worker_test.go#L153), and this test don't need to check pr chan

Reply:
I don't think this matters.